whois-query/src/main/java/net/ripe/db/whois/query/planner/RpslResponseDecorator.java

```diff
@@ -2,27 +2,27 @@
 
 import com.google.common.collect.Iterables;
 import com.google.common.collect.Sets;
-import net.ripe.db.whois.common.oauth.OAuthSession;
-import net.ripe.db.whois.common.sso.UserSession;
-import net.ripe.db.whois.common.x509.ClientAuthCertificateValidator;
 import net.ripe.db.whois.common.collect.IterableTransformer;
 import net.ripe.db.whois.common.dao.RpslObjectDao;
+import net.ripe.db.whois.common.dao.UserDao;
 import net.ripe.db.whois.common.domain.ResponseObject;
-import net.ripe.db.whois.common.x509.X509CertificateWrapper;
+import net.ripe.db.whois.common.oauth.OAuthSession;
 import net.ripe.db.whois.common.rpsl.RpslObject;
 import net.ripe.db.whois.common.rpsl.transform.FilterAuthFunction;
 import net.ripe.db.whois.common.rpsl.transform.FilterChangedFunction;
 import net.ripe.db.whois.common.rpsl.transform.FilterEmailFunction;
 import net.ripe.db.whois.common.source.SourceContext;
 import net.ripe.db.whois.common.sso.AuthServiceClient;
-import net.ripe.db.whois.common.sso.SsoTokenTranslator;
+import net.ripe.db.whois.common.sso.UserSession;
+import net.ripe.db.whois.common.x509.ClientAuthCertificateValidator;
+import net.ripe.db.whois.common.x509.X509CertificateWrapper;
 import net.ripe.db.whois.query.QueryMessages;
 import net.ripe.db.whois.query.domain.MessageObject;
 import net.ripe.db.whois.query.executor.decorators.DummifyDecorator;
 import net.ripe.db.whois.query.executor.decorators.FilterPersonalDecorator;
 import net.ripe.db.whois.query.executor.decorators.FilterPlaceholdersDecorator;
 import net.ripe.db.whois.query.query.Query;
-import org.apache.commons.lang3.StringUtils;
+import org.elasticsearch.common.Strings;
```

Comment:
Wrong import? We should use stringUtils

Reply:
We should only use the Apache commons library for this, not any "internal" string handling classes from other libraries, that is not it's main function. I agree we should only use elasticsearch packages for full text search.

We should migrate from lang(2) to lang3, currently *both* are Whois dependencies, because some lang3 methods throw NullPointerException which is an unwelcome change and we need to either expect that or write our own. As discussed I'll add a new task to migrate off lang(2).

Reply:
I updated this using StringUtils